### PR TITLE
Add torch fmin frontend

### PR DESCRIPTION
**Pull request by Lapatou**
- [ ] #3861

### CODE REVIEW

**Comment by kimbring2** [comment]
@Lapatou Hello, your code passes the test. However, you use too much line space. Please remove it!

**Comment by Lapatou** [comment]
@kimbring2 I formatted as your request please check.

**Comment by kimbring2** [comment]
@Lapatou Thank you (: I merge it

**Comment by Lapatou** [comment]
@kimbring2 Thank you to😊

File: ivy/functional/frontends/torch/comparison_ops.py [hunk 1]
```diff
@@ -168,3 +168,7 @@ def fmax(input, other, *, out=None):
         other,
         out=out,
     )
+
+
+def fmin(input, other, *, out=None):
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/torch/comparison_ops.py:173]: This function is supposed to handle nans but behaves differently from `torch.fmin` behavior.

Comment (kimbring2) [reply ivy/functional/frontends/torch/comparison_ops.py:173]: @hirwa-nshuti Hello, I am going to check it again. Thank you for notifying (:

**Comment by hirwa-nshuti** [comment]
I think I can have a quick fix for this can I make a PR

File: ivy/functional/frontends/torch/comparison_ops.py (hunk 1 above)
Comment (hirwa-nshuti) [reply ivy/functional/frontends/torch/comparison_ops.py:173]: Hi there.
I think I can have a quick fix for this, can I open a PR for the fix?

Comment (kimbring2) [reply ivy/functional/frontends/torch/comparison_ops.py:173]: @hirwa-nshuti Oh, you already fixed it. Yeah, please open the PR!

**Pull request merged**